Cargo.toml

```diff
@@ -5,6 +5,7 @@ license = "MIT"
 name = "conduit-cookie"
 repository = "https://github.com/conduit-rust/conduit-cookie"
 version = "0.9.0-alpha.5"
+edition = "2021"
```

Comment:
the rest of the conduit crates currently use a MSRV of v1.46.0, but if we declare `edition = 2021` here we'd need to bump that to v1.56.0. I'm not sure if we want to do that already.

what do you think about only moving to 2018 for now?

Reply:
> the rest of the conduit crates currently use a MSRV of v1.46.0

Oh, didn't know that, let's migrate to 2018 then 👍

Reply:
it wasn't explicitly declared so far, but I've added it to the main repos in the last few days/weeks. I've used https://github.com/foresterre/cargo-msrv to figure out what the implicit MSRV was.

Reply:
I guess we should probably do something similar to https://github.com/conduit-rust/conduit-hyper/pull/26 here too